bsdbpf/bsd_bpf_sniffer.go

```diff
@@ -0,0 +1,155 @@
+// +build darwin dragonfly freebsd netbsd openbsd
```

Comment:
Little nit:  could you move the copyright to the top and put this down below it?